scripts/src/javatests/org/oppia/android/scripts/testfile/TestFileCheckTest.kt

```diff
@@ -57,13 +60,7 @@ class TestFileCheckTest {
       "File ${retrieveTestFilesDirectoryPath()}/ProdFile2.kt does not have a corresponding test" +
         " file."
     )
-    assertThat(outContent.toString()).contains(
-      "If this is correct, please update scripts/assets/test_file_exemptions.textproto"
-    )
-    assertThat(outContent.toString()).contains(
-      "Note that, in general, all new files should have tests. If you choose to add an" +
-        " exemption, please specifically call this out in your PR description."
-    )
+    assertThat(outContent.toString()).contains(wikiReferenceNote)
```

Comment:
Shouldn't we also be checking that it's at the end of the output for this & the other test file checks?

Reply:
Done.